NetEase/arctic
# [ARCTIC-362][AMS]add schema sql file for every version for ams

## Why are the changes needed?
resolve #362 

## Brief change log
add schema sql file for every version for ams

## How was this patch tested?
test procedures:
1. use `0.3.0-init.sql` and `upgrade-sql` to database1. exec `mysql -h {mysql_host} -P {mysql_port} -u {user} -p {password} database1 < {AMS_HOME_DIR}/conf/mysql/0.3.0-init.sql` and `mysql -h {mysql_host} -P {mysql_port} -u {user} -p {password}database1 < {AMS_HOME_DIR}/conf/mysql//upgrade-0.3.0-to-0.3.1.sql`
2. use `0.3.1-init.sql` to database2. exec `mysql -h {mysql_host} -P {mysql_port} -u {user} -p {password} database2 < {AMS_HOME_DIR}/conf/mysql/0.3.1-init.sql`
3. Check whether the schemas of database1 and database2 are the same.

## Documentation

  - Does this pull request introduces a new feature? (no)
  - If yes, how is the feature documented? (docs)
